Resources/Prototypes/Entities/Constructible/Furniture/instruments.yml

```diff
@@ -35,6 +25,7 @@
     interfaces:
     - key: enum.InstrumentUiKey.Key
       type: InstrumentBoundUserInterface
+  - type: Climbable
```

Comment:
make instruments climbable? ok i guess?

Reply:
Yeah, only meant for the piano though. I updated it.